runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -237,8 +245,55 @@ public void addDestination(String logicalGroup, List<String> remoteDestinations)
         }
     }
 
+    /**
+     * Sets a list of destinations for a group to the LogReplicationModelMetadataTable for monitoring.
+     * <p>
+     * A new logical group will be created if one does not already exist.
+     * Setting destinations for a logical group will overwrite existing destinations.
+     * Setting destinations to an empty list not, i.e. "clearing", is not permitted.
+     *
+     * @param logicalGroup The group of tables to be replicated, represented as a string.
+     * @param remoteDestinations The clusters which the specified group will be replicated to.
+     */
+    public void setDestinations(String logicalGroup, List<String> remoteDestinations) {
```

Comment:
would be better to confirm with them, but I think the requested api is 
```
List<String> setDestinations(String logicalGroup, List<String> destination)
```
where the returned List is the updated list of destinations of given `logicalGroup`, not sure is the `addDestinations` api still needed